README.md

```diff
@@ -1,6 +1,64 @@
+[rfc-audience]: #rfc-audience
 
+The primary audience of RFCs is the TVM development community. RFCs serve as a
+guide for the design and implementation of features during and after their
+development. A secondary audience is general users and developers who are
+interested in how and why a feature was designed and implemented.
+
+## RFC Workflow
+[rfc-workflow]: #rfc-workflow
+
+- **Community Discussion**: A need or issue is brought to the discussion forum.
+  During this phase, the developer and user community can discuss the need for
+  and requirements of the RFC
+- **Pull Request**: After or concurrent with the conversation on the discussion
+  forum, a pull request is created using the format prescreibed by the
+  [RFC Template](https://github.com/apache/tvm-rfcs/blob/c436b06818cb65f72cac821616670b22a2473619/0000-template.md).
+    - Discussion about the details of the RFC can continue in the pull request.
+	- A committer of the corresponding area will approve and merge the RFC.
+      Normally the corresponding committer will become the shepherd of the
+      implementation PRs.
+	- RFCs are numbered consecutively based on their order of proposal,
+      regardless of if they are accepted or postponed. 
+	- A successful RFC will include an overview with the problem the RFC is
+      attempting to address, a proposed solution that describes the design and
+      implementation strategy, and a timeline for completion. Optional sections can
+      include (but are not limited to) alternatives that were considered, security
+      considerations, and open problems that the RFC does not solve.
+	- It is expected that RFCs will change, as part of the feedback process and
+      as new implementation details arise. Changes to the RFC should not be squashed
+      or force pushed in order to retain change and discussion history.
+- **Tracking Issue**: Upon merging a RFC, a tracking issue will be created where
+  implementors can continue sharing implementation details (including links to
+  pull requests). The issue will be closed when the RFC is either completed or
+  abandoned. 
+- **Implementation**: Work will begin on the RFC, with
+pull requests linking back to the tracking issue. Upon completion of the RFC,
+the tracking issue will be closed and the RFC will be moved to the
+docs/rfc/completed directory.
+- **Postponement**: An RFC may be postponed either
+  explicitly by the parties responsible for implementing it, or implicitly by
+  having no work done for a period of time defined by project leaders. The RFC
+  will be moved to the docs/rfc/postponed directory
+	- **Resuming an Postponed RFC**: Work on a postponed RFC may be resumed by a
+      new responsible party at any time after another discussion and pull request
+      review process to move the RFC from docs/rfc/abandoned to docs/rfc/active.
+
+## References
+[references]: #references
+
+[RFC Discussion Post](https://discuss.tvm.apache.org/t/rfc-update-rfc-process/9033)
```

Comment:
Moving forward, we need to use the docs under tha RFC as source of truth, so do not need to refer to the original discuss post.

Reply:
I agree, but I think it's still possible to reference forum posts from them? though RFCs should state the rationale behind their decisions, it may still be helpful to refer to the original discussion

Reply:
My thinking was that the RFC in the repository is the source of truth, but that there's value to linking back to the original discussion. There's a balance though, I didn't link back to the community meeting. I'm happy to follow whatever consensus is.

Reply:
My original thinking was mostly the other way around, the rfc doc suppose to distill everything available in the forum(so people dont have to lazily refer back), but we can for sure link to discussions when we open RFC PRs etc.

Reply:
Raised the question in the discussion forum. My preference is to just link back. Links are cheap, and it saves someone from having to run a search to find old context or reference reasoning behind particular decisions.

Reply:
@tqchen I think it would be great to avoid linking to design documentation in the forum (as a goal--this will be impossible in the first few RFCs which will need to reference those posted on past forum topics). However, I think that it's still worth allowing linking to reference e.g. community discussion to understand how each design conclusion was reached. So the line would be: if you're linking to something which describes a design, that needs to be another RFC in the repo (or a past RFC on Discuss which was not yet ported). If you're just linking to additional context around a decision, or to something directly evidenced in a forum post, that's ok. However, we preserve the original goal of fully describing a design in the GH RFCs.